### PR TITLE
Status card

### PR DESCRIPTION
**Pull request by sroy3**
https://github.com/coveo/vapor/pull/580 needed for a successful build.

### CODE REVIEW

File: src/components/statusCard/styles/StatusCard.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    background: $pure-white;
+
+    &:global(.simple) {
+        background: transparent;
+        box-shadow: none;
+    }
+}
+
+@each $color, $value in $palette {
+    &:global(.border-color-#{$color}) {
```
Comment (GermainBergeron) [inline src/components/statusCard/styles/StatusCard.scss]: This should probably go in https://github.com/coveo/vapor/blob/b952495364c87f5e057de405c7d8f5c7666cb8eb/scss/utility/border.scss

File: src/components/statusCard/styles/StatusCard.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
+@import '~coveo-styleguide/scss/common/palette.scss';
```
Comment (pochretien) [inline src/components/statusCard/styles/StatusCard.scss]: can we create an alias with this 3 imports ? if possible to do that. I think we will use these imports in many files

**Comment by sroy3** [comment]
Demo: https://coveo.github.io/react-vapor/status-card/index.html

File: src/components/statusCard/styles/StatusCard.scss (hunk 2 above)
Comment (sroy3) [reply src/components/statusCard/styles/StatusCard.scss]: I don't think this is possible. I didn't need the map anymore and we don't always want to import both other files.

**Pull request merged**